portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,82 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+var absolute_index = 0;
+//absolute_index is the index of the picture
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function loadIntro(){
+    window.location =("/intro.html");
```

Comment:
I'm surprised this works? I'd think you would need a relative url here (starting with a ".")?

Reply:
If it does what you want, that's OK, just want to make sure.